error_handling/go.mod

```diff
@@ -1,3 +1,10 @@
 module github.com/blocky/as-demo
 
-go 1.22
+go 1.22.6
+
+require github.com/blocky/basm-go-sdk v0.0.0-20250307220910-8acf608334a6
```

Comment:
interesting, this one is as expected.  I wonder if there is something in the workflow that needs to be addressed.